### PR TITLE
Feature/colors and conditional

### PR DESCRIPTION
**Pull request by andymcfee**
image

### CODE REVIEW

File: src/alto-ui/Table/Table.scss [hunk 1]
```diff
@@ -266,7 +238,7 @@
 
 .Table__cell-input {
   display: none;
-  margin: 0 3px;
+  margin: 0 0.25rem;
```
Comment (renaudtertrais) [inline src/alto-ui/Table/Table.scss]: `$spacing-x-small` ?

File: src/alto-ui/Table/Table.scss [hunk 2]
```diff
@@ -130,10 +127,14 @@
   height: 100%;
   white-space: nowrap;
   text-overflow: ellipsis;
-  padding: 0.5rem 1rem;
+  padding: 0.25rem 1rem;
```
Comment (renaudtertrais) [inline src/alto-ui/Table/Table.scss]: we should use `$spacing-x` unit no ?

File: src/alto-ui/Table/Table.scss [hunk 3]
```diff
@@ -93,35 +93,32 @@
...
+  // box-shadow: inset -6px 0 0 -1px $red;
+  // background: $red-10;
 }
 
 .Table__cell-container {
   width: 100%;
   height: 100%;
   display: flex;
   align-items: center;
+  // flex-direction: row-reverse;
```
Comment (renaudtertrais) [inline src/alto-ui/Table/Table.scss]: if it is not necessary anymore, you can remove it

File: src/alto-ui/Table/Table.scss [hunk 4]
```diff
@@ -93,35 +93,32 @@
   }
 }
 
+.Table__cell--editable {
+  padding-top: 0.25rem;
+  padding-bottom: 0.25rem;
+}
+
 .Table__cell--success {
```
Comment (NicolasMartos) [inline src/alto-ui/Table/Table.scss]: Do you need to keep this classes ?

**Pull request merged**